app/models/framework/definition/RM3786.fdl

```diff
@@ -0,0 +1,148 @@
+Framework RM3786 {
```

Comment:
I'm assuming this framework passes the validation task? You mentioned that they had in previous commit messages, so just making sure.

Reply:
Yes, I put all the framework names in a local `all_frameworks.txt` and ran

```
for FRAMEWORK in $(cat all_frameworks.txt); do echo $FRAMEWORK && rake fdl:validations:test[$FRAMEWORK,1000]; done
```

So they all pass for a sample of 1000 rows.  More than that means an overnight run :)